doc/scripts/BATCH.md

```diff
@@ -34,49 +34,48 @@ Parallelization with `brun_fastsurfer.sh`
 -----------------------------------------
 
 `brun_fastsurfer.sh` has powerful builtin parallel processing capabilities. These are hidden underneath the 
-`--parallel_subjects* <n>|max` and the `--device <device>` as well as `--viewagg_device <device>` flags.
+`--parallel* <n>|max` and the `--device <device>` as well as `--viewagg_device <device>` flags.
 One of the core properties of FastSurfer is the split into the segmentation (which uses Deep Learning and therefore 
 benefits from GPUs) and the surface pipeline (which does not benefit from GPUs). For ideal batch processing, we want 
 different resource scheduling.
 
-`--parallel_subjects*` allows three parallel batch processing modes: serial, single parallel pipeline and dual parallel
-pipeline. 
+`--parallel*` allows three parallel batch processing modes: serial, single parallel pipeline and dual parallel pipeline. 
 
 ### Serial processing (default)
 Each case/image is processed after the other fully, i.e. surface reconstruction of case 1 is fully finished before 
-segmentation of case 2 is started. This setting is the default and represents the manual flags `--parallel_subjects 1`.
+segmentation of case 2 is started. This setting is the default and represents the manual flags `--parallel 1`.
 
 ### Single parallel pipeline
 This mode is ideal for CPU-based processing for segmentation. It will process segmentations and surfaces in series 
 in the same process, but multiple cases are processed at the same time.
 
 ```bash
-$FASTSURFER_HOME/brun_fastsurfer.sh --parallel_subjects 4 --threads 2 --parallel
+$FASTSURFER_HOME/brun_fastsurfer.sh --parallel 4 --threads 2
```

Comment:
for same behaviour as before threads would need to be 4?

Reply:
Yes, but the text below is updated to reflect the behavior as "requested".